testing/run-scripts/run_cloud.sh

```diff
@@ -118,7 +118,7 @@ if ! docker ps -a | grep uproxy-zork >/dev/null; then
   then
       RUNARGS="$RUNARGS -p"
   fi
-  docker run --restart=always --net=host  $HOSTARGS --name uproxy-zork -d uproxy/$1 /test/bin/load-zork.sh $RUNARGS
+  docker run --restart=always --net=host  $HOSTARGS --name uproxy-zork -d uproxy/$1 /test/bin/load-zork.sh $RUNARGS -n
```

Comment:
Unconditionally passing `-n` will break some other features we use a lot, such as `-p`. Better tp add `-n` as a arg to this script, as we've done for `-p`. Sorry I know these args a pain :-/